platforms/Dos/Makefile

```diff
@@ -29,8 +29,9 @@ CXXFLAGS := $(CFLAGS) -xds
 
 .PHONY: all clean
 
-all: CPPU.LIB CPPUX.LIB CPPU1.EXE CPPU2.EXE CPPU3.EXE CPPU4.EXE \
-    CPPU5.EXE CPPU6.EXE CPPU7.EXE
+all: CPPU.LIB CPPUX.LIB \
+     CPPU1.EXE CPPU2.EXE CPPU3.EXE \
+     CPPUX1.EXE CPPUX2.EXE CPPUX3.EXE CPPUX4.EXE
 
```

Comment:
Oops, I only looked at the shell script, and it seemed to handle this fine.

I assume by batch file you mean ALLTESTS.BAT, to me it looks like it just runs any *.exe file in the current directory.